### DISCUSSION TITLE
uYou tabs

### DISCUSSION THREAD

**Discussion by yodaluca23** [qnblackcat/uYouPlus#578] [Q&A]
Does anyone know how to remove the uYou tab in the bottom right? I went to uYou settings but there is no option to remove the uYou tab only the vanilla ones... Also, rearranging tabs isn't working for me... I've tried restarting the app several times...

Also, does anyone have the Discord link?

**Comment by arichornlover** [comment]
You ain’t supposed to remove the uYou tab, if you remove the uYou tab then the uYou Settings will be gone.
Unless qnblackcat adds an option just like CercubePlus but I likely doubt it will happen.

**Reply by yodaluca23** [reply]
Well it should just be in the standard settings area
